releasenotes/notes/figure_return_SVG-4ad72fc8a3bee3cb.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Load experiment figures are now automatically viewable in Jupyter Notebook.
```

I would classify this as a bug fix because it was intended to work this way before:

```suggestion
fixes:
  - |
    Figures loaded from the experiment service are now rendered correctly in Jupyter Notebook.
```